Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -86,7 +86,7 @@ def from_string(self, string):
                 return self.issue(int(match.group('id')))
         return None
 
-    def credentials(self, required=True, validate=False):
+    def credentials(self, required=True, validate=False, save=None):
```

Comment:
This is ok, but I would prefer more explicit variable name like save_in_keychain, so it is clear to the engineer reading this code in future.